docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,336 @@
+:Number:BP000
+:Title: Blueprint specification and requirements
+:Author: Beraldo Leal <bleal@redhat.com>
+:Reviewers: Cleber Rosa <crosa@redhat.com>, Jan Richter <jarichte@redhat.com>,
+            Plamen Dimitrov <pdimitrov@pevogam.com>, Willian Rampazzo
+            <willianr@redhat.com>
+:Created: 29-Sep-2020
+:Type: Meta Blueprint
+:Status: Approved
+
+.. contents:: Table of Contents
+
+TL;DR
+*****
+
+Having better records of architectural decisions in our repository is a good
+way to socialize our technical decisions, improve the code review process,
+avoid unnecessary code, and balance the workload among other developers. We are
+already using the “blueprint” documents for a while and still learning how to
+write them. Although we have a basic idea of what is a blueprint, some may not
+understand the concept because we are missing one meta-document that describes
+the blueprint's basic notions. This document is a blueprint to specify how we
+should write blueprints from now on.
+
+Motivation
+**********
+
+Depending on the project size, having very well defined and structured
+documents about the architecture decisions seems like an overkill, but could
+help projects of any size, including Avocado, save time, make better decisions
+and improve the way we socialize those decisions.
+
+Today in the Avocado project we have the good practice to submit an RFC to
+our mailing list or to use/open a GitHub issue when we have a new idea. RFCs are a
+widespread way to disclose the architecture decisions, but they are just one
+part of a longer process. During the RFC phase, we argue in favor of a
+proposal, and we are mostly concerned about collecting feedback. After this
+phase, we could go one step forward and consolidate the discussion in
+Blueprints (sometimes called ADRs - Architecture Decision Records). This could
+be the next step so we could better socialize our decisions for future readers.
```

Comment:
"socialize" is a bit repetitive here, perhaps "so we could better communicate our decisions to future readers"?

Reply:
In this section, I'm using the word "socialize" twice. On in each paragraph. Yes, It was my intention to reinforce this idea.

I would like to keep it.

Reply:
Alright , it seems over-repetitive for me but your call, I guess you have a goal behind this.